src/main/java/carproject/controller/CarController.java

```diff
@@ -0,0 +1,48 @@
+package carproject.controller;
+
+import carproject.dto.CarRequestDto;
+import carproject.dto.CarResponseDto;
+import carproject.mapper.CarMapper;
+import carproject.model.Car;
+import carproject.service.CarDoorService;
+import carproject.service.CarService;
+import carproject.service.CarWheelService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/car")
+public class CarController {
+    @Autowired
+    private CarService carService;
+
+    @Autowired
+    private CarDoorService carDoorService;
+
+    @Autowired
+    private CarWheelService carWheelService;
+
+    @Autowired
+    private CarMapper carMapper;
+
+    @PostMapping("/add-new-car")
```

Comment:
```suggestion
    @PostMapping()
```
https://restfulapi.net/resource-naming/
давай змінимо неймінг згідно з best practices